src/App.js

```diff
@@ -32,6 +34,8 @@ function App() {
         <Route path="/wishlist" element={<WishListView />} />
         <Route path="/checkout/completed" element={<CheckoutComplete />} />
         <Route path="/profile" element={<UserProfile />} />
+        <Route path="/forgot-password" element={<ForgotPassword />} />
+        <Route path="/forgot-password" element={ResetPassword} />
```

Comment:
<img width="911" alt="Screenshot 2023-11-13 at 10 02 31 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/3c303754-8fac-4985-a87d-00ece52f825c">

`ResetPassword` by itself is the function returned from the `ResetPassword.js` file (`const ResetPassword = () => {...}` is a function). 

But `Route` component is not expecting a function in the `element` prop, it's expecting a _component_. To make a React function into a component, we need to add the `< ...>` brackets to render it as a DOM element. Note how all the other elements in the other routes are written.